Documentation/Conditions/Index.rst

```diff
@@ -1324,3 +1324,26 @@ usergroup()
     :ref:`condition-frontend-user` and :ref:`condition-backend-user` instead.
     For migration hints see the
     :ref:`changelog <ext_core:deprecation-100349-1680097287>`.
+
+
+..  _condition-examples:
+
+Examples
+========
+
+..  _condition-examples-constant:
+
+Check if a constant is set to a certain value
+---------------------------------------------
+
+TypoScript constants can be used in conditions with the 
+:ref:`Syntax <t3coreapi:typoscript-syntax-conditions-syntax>` for conditions:
+
+..  code-block:: typoscript
+    :caption: EXT:my_extension/Configuration/TypoScript/setup.typoscript
+
+    [{$tx_my_extension.settings.feature1Enabled} == 1]
```

Comment:
I love examples. Thank you. What about such information:

Works:
```
# constant
foo = 1

# setup
[{$foo} == 1]
```

Works:
```
# constant
foo = 1

# setup
[{$foo} == '1']
```

Works:
```
# constant
foo = 1

# setup
[{$foo} === 1]
```

Does not work:
```
# constant
foo = 1

# setup
[{$foo} === '1']
```

Works:
```
# constant
foo = bar

# setup
[{$foo} === 'bar']
```

Does not work:
```
# constant
foo = bar

# setup
[{$foo} === bar]
```

Does not work:
```
# constant
foo = bar

# setup
[{$foo} == bar]
```

Works:
```
# constant
foo = bar

# setup
[{$foo} == 'bar']
```

Works:
```
# constant
foo = bar

# setup
[{$foo} == "bar"]
```